src/markdown-view.android.ts

```diff
@@ -1,29 +1,36 @@
-import { markdownProperty, MarkdownViewBase } from './markdown-view.common';
-
+import { fontSizeProperty, markdownProperty, MarkdownViewBase } from './markdown-view.common';
+​
 declare var ru: any;
-
+​
 export class MarkdownView extends MarkdownViewBase {
-
+​
     _android: any = null; // android.widget.TextView
-
+​
     markwon: any;
-
+​
+    private _fontSize = 14;
```

Comment:
Is 14 on Android equivalent to 18 on iOS? If not, we should make the default font sizes the same.